src/NetworkTest/testQuality/helpers/subscriberMOS.ts

```diff
@@ -102,7 +107,7 @@ function calculateAudioScore(
       if (R > 100) {
         return 4.5;
       }
-      return 1 + (0.035 * R) + ((7.10 / 1000000) * R) * (R - 60) * (100 - R);
+      return 1 + (0.035 * R) + R * (R - 60) * (100 - R) * 0.000007;
```

Comment:
Same formula. I just re-write it to be more similar than in the ITU-T description.

Reply:
Good.